tasks/Sequential dyadic die-rolling.md

```diff
@@ -0,0 +1,112 @@
+# Summary
+> This study examines the the role of collaboration in shaping corruption through a sequential dyadic dice-rolling game. In a incentive structure based on rolling doubles, players in the collaborative enviornment reported signficantly more doubles than when players played alone and when lying only benefited one party. 
+
+
+# References
+> Appendix File includes instructions and computer screen reference images: https://www.pnas.org/content/pnas/suppl/2015/08/05/1423035112.DCSupplemental/pnas.1423035112.sapp.pdf
```

Comment:
I think we have to provide a citation as well. 

Weisel, O., & Shalvi, S. (2015). The collaborative roots of corruption. Proceedings of the National Academy of Sciences, 112(34), 10651-10656.